deployment/arm/deploy-region.sh

```diff
@@ -1,11 +1,24 @@
 #!/bin/bash
+set -eu
+
 parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$parent_path"
 
 businessUnit=$1
 appName=$2
 env=$3
 location=$4
+
+if [[ $# -ne 4 ]]; then
+    echo "Illegal number of arguments"
+    exit 1
+fi
+
+if [[ -z $businessUnit || -z $appName || -z $env || -z $location ]]; then
```

Comment:
is this needed?

Reply:
I was debating on this to keep it or not. But i think it makes more sense to remove it